res/effects_menu_defaults.xml

```diff
@@ -0,0 +1,93 @@
+<EffectMenuList>
+    <Group>
+        <Name>Volume and Compression</Name>
+        <Effects>
+            <Effect>Amplify</Effect>
+            <Effect>Compressor</Effect>
+            <Effect>Limiter</Effect>
+            <Effect>Normalize</Effect>
+            <Effect>Loudness Normalization</Effect>
+            <Effect>Auto Duck</Effect>
+        </Effects>
+    </Group>
+    <Group>
+        <Name>Fading</Name>
+        <Effects>
+            <Effect>Fade In</Effect>
+            <Effect>Fade Out</Effect>
+            <Effect>Studio Fade Out</Effect>
+            <Effect>Adjustable Fade</Effect>
+            <Effect>Crossfade Clips</Effect>
+            <Effect>Crossfade Tracks</Effect>
+        </Effects>
+    </Group>
+    <Group>
+        <Name>Pitch and tempo</Name>
```

Comment:
Which capitalization convention should we use?  This is not like "Volume and Compression".  Should it have "Tempo"?